### ISSUE TITLE
incorrect base image

### ISSUE THREAD

**Issue by atawfeek** [Azure-Samples/sftp-creation-template#26]
The current used base image _microsoft/azure-cli:latest_ is no longer available
The incorrect base image causes failure during deployment showing the following error message:
`{"code":"DeploymentFailed","message":"At least one resource deployment operation failed. Please list deployment operations for details. Please see https://aka.ms/DeployOperations for usage details.","details":[{"code":"InaccessibleImage","message":"The image 'microsoft/azure-cli:latest' in container group 'create-share-group' is not accessible. Please check the image and registry credential."}]}`

Should be replaced with _mcr.microsoft.com/azure-cli:latest_

**Comment by armanatory** [comment]
Same problem.

**Comment by atawfeek** [comment]
Should be replaced with mcr.microsoft.com/azure-cli:latest

Tested and works by editing the template on Azure during deployment.

**Comment by amitbhatik** [comment]
After fixing the Azure CLI Image it stuck with SFTP image. 


  "code": "DeploymentFailed",
  "message": "At least one resource deployment operation failed. Please list deployment operations for details. Please see https://aka.ms/DeployOperations for usage details.",
  "details": [
    {
      "code": "InaccessibleImage",
      "message": "The image 'atmoz/sftp:latest' in container group 'sftp-group' is not accessible. Please check the image and registry credential."
    }
  ]

Not sure what exactly the cause of this.

**Comment by armanatory** [comment]
I have the same error on atmoz/sftp:latest after fixing mcr.microsoft.com/azure-cli:latest.

**Comment by tomforeman** [comment]
Experienced the same InaccessibleImage issue today with atmoz/sftp:latest after fixing mcr.microsoft.com/azure-cli:latest.

Managed to circumvent the InaccessibleImage issue for atmoz/sftp by also updating the tag in the 'sftpContainerImage' variable from latest to debian in the azuredeploy.json templates.

**My changes:**
```
"cliContainerImage": "mcr.microsoft.com/azure-cli:latest"
"sftpContainerImage": "atmoz/sftp:debian"
```

**More info on atmoz/sftp packages:**
https://github.com/atmoz/sftp

Hope this helps somebody in the meantime.

**Comment by armanatory** [comment]
> Experienced the same InaccessibleImage issue today with atmoz/sftp:latest after fixing mcr.microsoft.com/azure-cli:latest.
> 
> Managed to circumvent the InaccessibleImage issue for atmoz/sftp by also updating the tag in the 'sftpContainerImage' variable from latest to debian in the azuredeploy.json templates.
> 
> **My changes:**
> 
> ```
> "cliContainerImage": "mcr.microsoft.com/azure-cli:latest"
> "sftpContainerImage": "atmoz/sftp:debian"
> ```
> 
> **More info on atmoz/sftp packages:**
> https://github.com/atmoz/sftp
> 
> Hope this helps somebody in the meantime.

Yeah it works

**Comment by bhummerstone** [comment]
Thanks folks! I'll see about getting these changes merged in. Unsure as to why the atmoz/sftp image isn't working with the latest tag, but I'll make that change too

**Comment by bhummerstone** [comment]
The Microsoft CLI fix has been merged in PR #25: thanks @BethFleming for that.

It's actually possible to use native ARM to create the file share instead of the Azure CLI, so I will update the template with this functionality instead
